### ISSUE TITLE
Data download

### ISSUE THREAD

**Issue by j-phi** [ioam/scipy-2017-holoviews-tutorial#14]
So it could absolutely just be me misunderstanding the instructions, but when I ran 'python download_sample_data.py' from the root of the clone of this repo (in my case, that's ./scipy-2017-holoviews-tutorial), the nyc_taxi.csv file was downloaded to the ./scipy-2017-holoviews-tutorial/data/ directory and it looks like it needs to be in the ./scipy-2017-holoviews-tutorial/notebooks/assets/ directory.

**Comment by jlstevens** [comment]
> ... it looks like it needs to be in the ./scipy-2017-holoviews-tutorial/notebooks/assets/ directory.

I don't think we expect the data to go there and the ``./scipy-2017-holoviews-tutorial/data/`` location does seems correct to me. What did you see that suggested the .csv file ought to be in ``assets``?

**Comment by j-phi** [comment]
In /notebooks/00-welcome.ipynb just above **Recommended Configuration**:

`import os
if not os.path.isfile('./assets/nyc_taxi.csv'):
    print('Taxi dataset not found.')`

**Comment by philippjfr** [comment]
Thanks @j-phi, I believe the actual notebooks look for it in ``./data`` so we'll update that asap.
